security/compliance_operator/compliance-operator-installation.adoc

```diff
@@ -0,0 +1,140 @@
+[id="compliance-operator-installation"]
+= Installing the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-operator-installation
+
+toc::[]
+
+
+Before you can use the Operator, you must ensure it is deployed in the cluster.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+. Create the target namespace:
++
+[source,terminal]
+----
+$ oc create -f deploy/ns.yaml
+----
+
+. Install the Operator from the web console or from the CLI:
++
+[source,terminal]
+----
+$ oc create -f deploy/olm-catalog/operator-group.yaml
+----
++
+[source,terminal]
+----
+$ oc create -f deploy/olm-catalog/subscription.yaml
+----
++
+The subscription file can be edited to optionally deploy a custom version. See the `startingCSV` attribute in the `deploy/olm-catalog/subscription.yaml` file.
+
+. Verify that the expected objects are created:
++
+[source,terminal]
+----
+$ oc get sub -n openshift-compliance
+----
++
+[source,terminal]
+----
+$ oc get ip -n openshift-compliance
+----
++
+[source,terminal]
+----
+$ oc get csv -n openshift-compliance
+----
++
+At this point, the Operator should be up and running:
++
+[source,terminal]
+----
+$ oc get deploy -n openshift-compliance
+----
++
+[source,terminal]
+----
+$ oc get pods -n openshift-compliance
+----
+
+To deploy from source:
```

Comment:
Let's not suggest customers to deploy from source; While it's possible, this is not something we're willing to support as the code might be bleeding edge.